launcher/Application.cpp

```diff
@@ -899,7 +926,8 @@ bool Application::createSetupWizard()
             return true;
         return false;
     }();
-    bool wizardRequired = javaRequired || languageRequired;
+    bool pasteInterventionRequired = settings()->get("PastebinURL") != "";
```

Comment:
Will this ever happen? The constructor of `Application` will already migrate existing settings.

Reply:
> Will this ever happen? The constructor of `Application` will already migrate existing settings.

See for yourself. The auto migration migrates people only if they had not used the default paste URL previously. This is triggered if the user had the default paste URL, and then after the manual migration choice is made the PastebinURL is reset meaning it won't trigger subsequently.

Reply:
Brain lag. Makes sense